### PR TITLE
Updated references of 'apt' to 'apt-get'

### PR DESCRIPTION
**Pull request by kichalla**

### CODE REVIEW

File: images/build/Dockerfile [hunk 1]
```diff
@@ -264,9 +265,6 @@ RUN chmod a+x /opt/buildscriptgen/GenerateBuildScript
 FROM python AS final
 WORKDIR /
 
-# Install rsync
-RUN apt update && apt install rsync --yes
```
Comment (dorfire) [inline images/build/Dockerfile]: :+1:

File: images/runtime/node/8.8/Dockerfile [hunk 2]
```diff
@@ -81,8 +81,8 @@ CMD [ "node" ]
 
 FROM node-original-image as oryx
 
-RUN  if command -v apt; then \
-       apt update && apt upgrade --yes; \
+RUN  if command -v apt-get; then \
```
Comment (dorfire) [inline images/runtime/node/8.8/Dockerfile]: Do we really need this `command` test? Isn't this image **always** debian-based?

File: images/runtime/node/8.2/Dockerfile [hunk 3]
```diff
@@ -80,8 +80,8 @@ CMD [ "node" ]
 
 FROM node-original-image as oryx
 
-RUN  if command -v apt; then \
-       apt update && apt upgrade --yes; \
+RUN  if command -v apt-get; then \
```
Comment (dorfire) [inline images/runtime/node/8.2/Dockerfile]: Same as my other comment

File: images/runtime/node/8.8/Dockerfile (hunk 2 above)
Comment (kichalla) [reply images/runtime/node/8.8/Dockerfile]: I do not know actually. Let's fix it in a separate PR if this we really care about it.

**Pull request merged**